.github/workflows/ci-react.yml

```diff
@@ -0,0 +1,56 @@
+name: CI (React)
+
+defaults:
+  run:
+    working-directory: react
+
+on:
+  push:
```

Comment:
Want to limit to push on `main` branch rather than every single push?

Reply:
sure